mindbender/maya/pythonpath/userSetup.py

```diff
@@ -2,6 +2,50 @@
 
 from maya import cmds
 
+import os
+
+
+def mayafpsconverter(Sfps):
```

Comment:
Ok, this should do it.

```python
def mayafpsconverter(fps):
  return {
    "15": "game",
    "24": "film",
    "25": "pal",
    "30": "ntsc",
    "48": "show",
    "50": "palf",
    "60": "ntscf",
    None: "",
    "": "",
  }.get(fps, fps + "fps")
```

In fact, I would either make this function an official part of the API, like put it in `mindbender.lib` or not make a function out of it.